exporter/signalfxexporter/translation/default_metrics.go

```diff
@@ -134,6 +134,11 @@ exclude_metrics:
   - system.network.dropped
   - system.network.tcp_connections
 
+# Virtual memory metrics.
```

Comment:
I believe we should also add `system.paging.operations` with `type: minor`, right?

Reply:
Yeah, good point! Added.

Reply:
Should we also add the following?

```
- metric_name: system.paging.operations
  dimensions:
    type: minor
```